experiments/acic18/script.R

```diff
@@ -3,6 +3,7 @@ set.seed(1)
 rm(list = ls())
 
 library(grf)
+if(packageVersion("grf") >= '0.10.2') {warning("This script requires grf 0.10.2 or higher");}
```

Comment:
style nitpick: could you please put the warning on its own line, indented (and without the semicolon)

Reply:
Sure thing!